### PR TITLE
Auth servers concept clarity improvement

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** Auth servers concept clarity improvement
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-666914](https://oktainc.atlassian.net/browse/OKTA-666914)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/auth-servers/index.md](https://github.com/okta/okta-developer-docs/blob/d6a89b19dfd38b102bde2a3181ce6fe5991ab592/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Fauth-servers%2Findex.md) | [93](https://okta.acrolinx.cloud/api/v1/checking/scorecards/47be9594-4563-4b29-9c48-6e58e35741d7) | 86  | 93  | 100  | 96  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.e9553774-5935-47cb-bb56-7003cc957d9d?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/concepts/auth-servers/index.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ meta:
 
 <ApiAmProdWarning />
 
-Authentication and authorization are essential to application development. Whether you're developing an internal IT app for your employees, building a portal for your partners, or exposing a set of APIs for developers building apps around your resources, you need the right authentication and authorization support for your projects. With Okta, you can control access to your application using both [OAuth 2.0 and OpenID Connect](/docs/concepts/oauth-openid/). Use Okta as your authorization server to retain all of your user information and grant users tokens to control their authorization and authentication.
+Authentication and authorization are essential to app development. Projects such as building a protal for your partners or developing an internal IT app for your employees need the right authentication and authorization support. With Okta, you can control access to your app using both [OAuth 2.0 and OpenID Connect](/docs/concepts/oauth-openid/). Use Okta as your authorization server to retain all of your user information and grant users tokens to control their authorization and authentication.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/concepts/auth-servers/index.md:12]:
```suggestion
Authentication and authorization are essential to app development. Projects such as building a portal for your partners or developing an internal IT app for your employees need the right authentication and authorization support. With Okta, you can control access to your app using both [OAuth 2.0 and OpenID Connect](/docs/concepts/oauth-openid/). Use Okta as your authorization server to retain all of your user information and grant users tokens to control their authorization and authentication.
```

**Review by brentschaus-okta — DISMISSED** [review]
LGTM

small nit

**Review by grahamsmith-okta — DISMISSED** [review]

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/auth-servers/index.md](https://github.com/okta/okta-developer-docs/blob/559e390ee5b97031d95c9a6a7c5f79ce47def26a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Fauth-servers%2Findex.md) | [94](https://okta.acrolinx.cloud/api/v1/checking/scorecards/79dc2d5a-2e7d-4823-ab1f-22c23c11046c) | 86  | 100  | 100  | 96  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.0b53cfc8-9178-41b5-bd65-316a8ef791aa?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by grahamsmith-okta — APPROVED** [review]

**Pull request merged**